internal/pkg/docker/dockercompose/decompose.go

```diff
@@ -7,6 +7,7 @@ import (
 	"fmt"
 	"github.com/compose-spec/compose-go/loader"
 	compose "github.com/compose-spec/compose-go/types"
+	"github.com/mitchellh/mapstructure"
```

Comment:
can you run `go mod tidy`? i think there should be a `go.mod` update...

Reply:
`mapstructure` is actually a dependency of `compose-go` so in a way we get it for "free" without having to add a new dependency in this PR!

Reply:
right, but it should still show up in `go.mod` when you explicitly use it 🤔